awx/ui_next/src/screens/User/User.jsx

```diff
@@ -1,121 +1,94 @@
-
-    this.setState({ contentError: null, hasContentLoading: true });
-    try {
-      const { data } = await UsersAPI.readDetail(id);
-      setBreadcrumb(data);
-      this.setState({ user: data });
-    } catch (err) {
-      this.setState({ contentError: err });
-    } finally {
-      this.setState({ hasContentLoading: false });
+function User({ i18n, setBreadcrumb }) {
+  const location = useLocation();
+  const match = useRouteMatch('/users/:id');
+  const userListUrl = `/users`;
+  const {
+    result: user,
+    error: contentError,
+    isLoading,
+    request: fetchUser,
+  } = useRequest(
+    useCallback(async () => {
+      const { data } = await UsersAPI.readDetail(match.params.id);
+      return data;
+    }, [match.params.id]),
+    null
+  );
+
+  useEffect(() => {
+    fetchUser();
+  }, [fetchUser, location.pathname]);
+
+  useEffect(() => {
+    if (user) {
+      setBreadcrumb(user);
     }
-  }
-
-  render() {
-    const { location, match, i18n } = this.props;
-
-    const { user, contentError, hasContentLoading, isInitialized } = this.state;
-
-    const tabsArray = [
-      { name: i18n._(t`Details`), link: `${match.url}/details`, id: 0 },
-      {
-        name: i18n._(t`Organizations`),
-        link: `${match.url}/organizations`,
-        id: 1,
-      },
-      { name: i18n._(t`Teams`), link: `${match.url}/teams`, id: 2 },
-      { name: i18n._(t`Access`), link: `${match.url}/access`, id: 3 },
-      { name: i18n._(t`Tokens`), link: `${match.url}/tokens`, id: 4 },
-    ];
-
-    let cardHeader = (
-      <TabbedCardHeader>
-        <RoutedTabs tabsArray={tabsArray} />
-        <CardActions>
-          <CardCloseButton linkTo="/users" />
-        </CardActions>
-      </TabbedCardHeader>
-    );
-
-    if (!isInitialized) {
-      cardHeader = null;
-    }
-
-    if (location.pathname.endsWith('edit')) {
-      cardHeader = null;
-    }
-
-    if (!hasContentLoading && contentError) {
-      return (
-        <PageSection>
-          <Card>
-            <ContentError error={contentError}>
-              {contentError.response.status === 404 && (
-                <span>
-                  {i18n._(`User not found.`)}{' '}
-                  <Link to="/users">{i18n._(`View all Users.`)}</Link>
-                </span>
-              )}
-            </ContentError>
-          </Card>
-        </PageSection>
-      );
-    }
-
+  }, [user, setBreadcrumb]);
+
+  const tabsArray = [
+    { name: i18n._(t`Details`), link: `${match.url}/details`, id: 0 },
+    {
+      name: i18n._(t`Organizations`),
+      link: `${match.url}/organizations`,
+      id: 1,
+    },
+    { name: i18n._(t`Teams`), link: `${match.url}/teams`, id: 2 },
+    { name: i18n._(t`Access`), link: `${match.url}/access`, id: 3 },
+    { name: i18n._(t`Tokens`), link: `${match.url}/tokens`, id: 4 },
+  ];
+
+  if (contentError) {
     return (
-      <PageSection>
-        <Card>
-          {cardHeader}
+      <Card>
```

Comment:
We need to wrap this in a <PageSection /> component, otherwise it will render like this:
<img width="813" alt="Screen Shot 2020-04-17 at 12 27 19 PM" src="https://user-images.githubusercontent.com/15881645/79591649-cec27b80-80a6-11ea-99e4-a999737783b7.png">